leaflet-draw/build.boot

```diff
@@ -1,34 +1,36 @@
 (set-env!
   :resource-paths #{"resources"}
-  :dependencies '[[cljsjs/boot-cljsjs "0.5.1" :scope "test"]])
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.1" :scope "test"]
+                  [cljsjs/leaflet "0.7.7-4"]])
 
 (require '[cljsjs.boot-cljsjs.packaging :refer :all])
 
 (def +lib-version+ "0.2.3")
-(def +version+ (str +lib-version+ "-1"))
+(def +version+ (str +lib-version+ "-2"))
 
 (task-options!
- pom  {:project     'cljsjs/leaflet-draw
-       :version     +version+
-       :description "Adds support for drawing and editing vectors and markers on Leaflet maps"
-       :url         "https://github.com/Leaflet/Leaflet.draw"
-       :scm         {:url "https://github.com/cljsjs/packages"}
-       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+  pom  {:project     'cljsjs/leaflet-draw
+        :version     +version+
+        :description "Adds support for drawing and editing vectors and markers on Leaflet maps"
+        :url         "https://github.com/Leaflet/Leaflet.draw"
+        :scm         {:url "https://github.com/cljsjs/packages"}
+        :license     {"MIT" "http://opensource.org/licenses/MIT"}})
 
 (require '[clojure.java.io :as io])
 
 (deftask package []
   (comp
-    (download :url      (str "https://github.com/Leaflet/Leaflet.draw/archive/master.zip")
-              :checksum "C2316EBEDD093B67F5F7083F75097444"
+    (download :url      (str "https://github.com/Leaflet/Leaflet.draw/archive/0.2.3.zip")
```

Comment:
It is a best practice to use `format` to create the url using `+lib-version+` here, like this: https://github.com/cljsjs/packages/blob/master/airbrake/build.boot#L28